src/pages/[id].page.jsx

```diff
@@ -70,7 +70,10 @@ export default function Details({ details }) {
             <SpecificationBlock
               key={block.id}
               heading={block.heading}
-              rowStart={masonryDetails[slug][block.heading]?.rowStart ?? 'auto'}
+              rowStart={
+                masonryDetails[slug][block.heading]?.rowStart(block.content)
+                ?? 'auto'
```

Comment:
This is tricky to know where all these variables are coming from! What is `auto` here?